internal/controller/controllers/common.go

```diff
@@ -47,7 +47,7 @@ const (
 	BackupLabelValue                 = "true"
 )
 
-//go:generate mockgen -package=controllers -destination=mock_k8s_client.go . K8sClient
+//go:generate mockgen --build_flags=--mod=mod -package=controllers -destination=mock_k8s_client.go . K8sClient
```

Comment:
can we add this build_flag more globally to the ``go generate`` command?
e.g. ``go generate --mod=mod ...``

Reply:
Not sure. The doc says https://github.com/golang/mock#debugging-errors

Reply:
Added build_flag globally to the `go generate` command